src/shellTokenize.test.js

```
@@ -0,0 +1,280 @@
+);
+
+assertTokens(
+  'POSIX: trailing backslash (no char follows) preserved',
+  'abc\\',
+  false,
+  ['abc\\'],
+);
+
+// ─── Summary ────────────────────────────────────────────────────────────────
+
+console.log(`\n${passed + failed} tests: ${passed} passed, ${failed} failed`);
```

ESLint

[error] Unexpected console statement.

(no-console)